### PR TITLE
Add Qidao Base

### PR DESCRIPTION
**Pull request by melotik**
Add QiDao Base

https://okgraph.xyz/?q=QmeivVyBQ8DpY5zBnzgHviy8fS5UnuCaVBNXVPUvyEKc7G

### CODE REVIEW

File: subgraphs/compound-forks/src/mapping.ts [hunk 1]
```diff
@@ -1642,7 +1642,7 @@ export function updateMarket(
     );
   }
 
-  const interestAccumulatedUSD = interestAccumulatedMantissa
+  const interestAccumulatedUSD = `interestAccumulatedMantissa`
```
Comment (dhruv-chauhan) [inline subgraphs/compound-forks/src/mapping.ts:1645]: typo?

File: deployment/deployment.json [hunk 2]
```diff
@@ -4753,6 +4753,32 @@
           }
         }
       },
+      "qidao-base": {
+        "network": "base",
+        "status": "prod",
+        "versions": {
+          "schema": "1.3.0",
+          "subgraph": "1.0.2",
```
Comment (dhruv-chauhan) [inline deployment/deployment.json:4761]: shouldn't the subgraph version be 1.0.0?

**Review by steegecs — APPROVED** [review]
Getting some linting errors in `constant.ts`.

File: deployment/deployment.json (hunk 2 above)
Comment (melotik) [reply deployment/deployment.json:4761]: Yeah, it's weird now with only being able to deploy to the studio. You can't redeploy the same version, so I had to increment each time i tested

**Pull request merged**